async-http-faraday.gemspec

```diff
@@ -10,13 +10,11 @@ Gem::Specification.new do |spec|
 	spec.summary       = "Provides an adaptor between async-http and faraday."
 	spec.homepage      = "https://github.com/socketry/async-http"
 
-	spec.files         = `git ls-files -z`.split("\x0").reject do |f|
-		f.match(%r{^(test|spec|features)/})
-	end
```

Comment:
I committed this in error 😞 I'm using the gem in a Docker container without git -- sorry about that.

Reply:
It’s okay I reverted it :)